test/component_test.exs

```diff
@@ -7,15 +7,15 @@ defmodule MozartFetcher.ComponentTest do
 
   describe "fetch components" do
     test "it returns the response body when succesfull" do
-      config = %Config{endpoint: "http://localhost:8082/success"}
+      config = %Config{endpoint: "http://localhost:8082/success", id: "123"}
```

Comment:
Ha I guess these technically are string now 😅

I think as Ettore mentioned in his comment that for documentation purposes it would be better to have the ids in the tests look like "real" ids. eg `"news_top_stories"`, `"news_navigation"`, or `"weather_forecast"` etc. rather than just a number

Reply:
ohhh I see, sorry that completely slipped my mind as I done it quickly to begin with! Ill sort that now :)